test/backends/zfs.sh

```diff
@@ -0,0 +1,27 @@
+zfs_setup() {
+  local LXD_DIR
+  LXD_DIR=$1
+
+  if ! which zfs >/dev/null 2>&1; then
+    echo "couldn't find zfs binary"; false
+  fi
+
+  truncate -s 100G "${LXD_DIR}/zfspool"
+  # prefix lxdtest- here, as zfs pools must start with a letter, but tempdir
+  # won't necessarily generate one that does.
+  zpool create lxdtest-$(basename "${LXD_DIR}") "${LXD_DIR}/zfspool"
```

Comment:
I don't know, but we're taking `basename` here, so we're really only getting 3 characters, since the tempdirs have the form `tmp.XXX/XXX`